script/release/uploaders/upload.py

```diff
@@ -83,6 +84,10 @@ def main():
     pdb_zip = os.path.join(OUT_DIR, PDB_NAME)
     shutil.copy2(os.path.join(OUT_DIR, 'pdb.zip'), pdb_zip)
     upload_electron(release, pdb_zip, args)
+  elif PLATFORM == 'linux':
```

Comment:
Do we generate a `debug.zip` for non-x64 platforms? (ia32, arm, arm64?)

Reply:
Nop, why ?

Reply:
In fact how can I access this info ? Thx!

Reply:
Done